gangplank/spec/cli.go

```diff
@@ -16,7 +16,7 @@ const (
 	fedoraGitRef = "testing-devel"
 
 	rhcosGitURL = "https://github.com/openshift/os"
-	rhcosGitRef = "master"
```

Comment:
I assume we want to do this?

Reply:
Yeah, it won't hurt.

Reply:
The gangplank code path here is not intended for production use so the change shouldn't break anything important. Further, for the Gangplank use case in Prow, it will be run as part of a Custom-Build strategy so Prow will provide these repos via the build API.

Reply:
Thanks for clarifying!